CUJExecutor/executeCuj

```diff
@@ -49,11 +49,11 @@ adb logcat -c
 if [ "$RECORD" = "r" ] 
 then 
 	RECSTART=$(( $(adb shell date '+%s%N') / 1000000))
-	adb shell "screenrecord /sdcard/test.mp4" &
+	adb shell "screenrecord --bugreport /sdcard/test.mp4" &
```

Comment:
Does `--bugreport` add significant overhead?

Reply:
doesn't seem like it, but will see. You referring to overhead before, during or after recording?

Reply:
There should be some overhead from overlaying numbers into the video.  Can manifest itself as higher CPU load, in turn slowing down the CUJ itself.

Reply:
oh yeah wow it tanks the fps of the video by a factor of ~5